package.json

```diff
@@ -23,7 +23,7 @@
     "LICENSE"
   ],
   "scripts": {
-    "build": "rimraf lib && tsc -d -p tsconfig.json",
+    "build": "rimraf lib && tsc -d -p src",
```

Comment:
Since I added a tsconfig.json file under `src`, I target the `src` project here. Later on, would we want to build the test suite, we can add a `tsconfig.json` under `tests` and target the `tests` project to build the test suite. This way, entry point can be different between the main and the tests projects.